recipe/meta.yaml

```
@@ -21,7 +21,11 @@ requirements:
     - python {{ python_min }}
 
   run:
+    - __{{ noarch_platform }}
     - python >={{ python_min }}
+    {% if noarch_platform == 'linux' %}
+    - libopengl-devel
```

Looks like there is:

```suggestion
    - libopengl
```

See: https://github.com/conda-forge/libglvnd-feedstock/blob/62c376495618d8ac04c90fdae9c058b138a4706e/recipe/meta.yaml#L65